### PR TITLE
Add accumulate

### PR DESCRIPTION
**Pull request by tongkiat**
For now, this only works for Guile.

I did not change `config.json` and `Makefile`.

### CODE REVIEW

File: exercises/accumulate/accumulate.scm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+(define accumulate
```
Comment (jitwit) [inline exercises/accumulate/accumulate.scm:1]: The skeleton should be a function having same argument names as the example, `f` and `xs`

**Comment by jitwit** [comment]
Overall looks good!
I'd suggest making it clear(er) that this won't work for Chez Scheme in the README only Guile. 
It might also be nice if there were a few more test cases. Perhaps something with nested lists to ensure the solution is mapping only the outermost layer?

**Comment by tongkiat** [comment]
I made the changes you suggested.  Please review.  Thanks.

**Comment by jitwit** [comment]
Perfect, thanks!

**Pull request merged**